docs/2-Deploying/2-Configuration.md

```diff
@@ -1240,6 +1240,73 @@ AWS STS role ARN to use for S3 authentication.
   	sts_assume_role_arn: arn:aws:iam::1234567890:role/some-s3-role
   ```
 
+##### Vulnerabilities
+
+###### `databases_path`
+
+The path specified needs to exist and fleet needs to be able to read and write to and from it. This is the only mandatory configuration needed for vulnerability processing to work.
+
+- Default value: none
+- Environment variable: `FLEET_VULNERABILITIES_DATABASES_PATH`
+- Config file format:
+
+  ```
+  vulnerabilities:
+  	databases_path: /some/path
+  ```
+
+###### `vulnerabilities_periodicity`
```

Comment:
All of these are actually without the `vulnerabilities_` prefix. The env vars have it because this is part of the `vulnerabilities` section in the yaml.